js/index.js

```diff
@@ -1 +1,61 @@
 // Your code goes here
+
+const navLinks = document.querySelectorAll('.nav-link');
+navLinks[0].addEventListener('click', (e) => e.preventDefault())
+navLinks[1].addEventListener('click', (e) => e.preventDefault())
+navLinks[2].addEventListener('click', (e) => e.preventDefault())
+navLinks[3].addEventListener('click', (e) => e.preventDefault())
+
+
+const intro = document.querySelector('.intro');
+intro.addEventListener('mouseenter', (e) => {
+    intro.style.border = '2px solid blue'
+})
+
+intro.addEventListener('mouseleave', () => {
+    intro.style.border = ''
+})
+
+window.addEventListener('scroll', () => {
+    const scrolled = window.scrollY;
+    console.log(scrolled);
+    if (Math.floor(scrolled) === 1016)
+    alert ('You\'ve reached the bottom!')
+})
+
+window.addEventListener('keydown', (e) => console.log(e.code))
+
+const headerLogo = document.querySelector('.logo-heading');
+headerLogo.addEventListener('wheel', () => {
+    headerLogo.style.color = 'yellow'
+    headerLogo.style.fontSize = '80px'
+})
+
+
+const destination = document.querySelector('.content-pick');
+
+destination.addEventListener('click', (e) => destination.style.backgroundColor = 'red');
+
+const button = document.querySelectorAll('.btn');
+
+button[0].addEventListener('click', (e) => {
+    button[0].textContent = "DON'T PRESS ME!";
+    e.stopPropagation();
+});
```

Comment:
If you just want one, you could use querySelector and grab the specific button you're interested in!